### DISCUSSION TITLE
How to enrich / add text in a simple way an annotation ?

### DISCUSSION THREAD

**Discussion by tristanbes** [community/community#25064] [Actions]
<p>Hello,</p>
<p>I wanted to know if it’s possible in a simple way to add more information when a task fails to the “Annotation” popin.</p>
<p>Right now, if a run fails, it simply displays the error like Process completed with exit code 1.but you lack of “quick context”</p>
<p>Example of task: </p>
<pre><code>- name: Runs phpspec
      run: |
        php -v
        php bin/phpspec run -f pretty
      if: always()
</code></pre>
<p>What I want to display in the “Annotation” popin:</p>
<p><em>“Run phpspec” (= task name) task failed, did you follow the internal guide ?</em></p>
<p>Here an example in the screenshot of how “annotation popin” lack of context for “run” command out of the box.</p>
<p>Thanks</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/0ce07f832bd5ebb90263994ef68f16504b754326.png" data-download-href="/uploads/short-url/1PUIiUhKFaJ8ePa0WP6b4lSLDnM.png?dl=1" title="upgrade_php_by_tristanbes_·Pull_Request__790·Yproximite_condictio_et_GitHub_Marketplace·_Actions_to_improve_your_workflow.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/346010/181088634-e62673e7-b963-4606-a316-160a4488c9ab.png" alt="upgrade_php_by_tristanbes_·Pull_Request__790·Yproximite_condictio_et_GitHub_Marketplace·_Actions_to_improve_your_workflow.png" data-base62-sha1="1PUIiUhKFaJ8ePa0WP6b4lSLDnM" width="546" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/0ce07f832bd5ebb90263994ef68f16504b754326_2_546x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/0ce07f832bd5ebb90263994ef68f16504b754326_2_819x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/0ce07f832bd5ebb90263994ef68f16504b754326.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/0ce07f832bd5ebb90263994ef68f16504b754326_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">upgrade_php_by_tristanbes_·Pull_Request__790·Yproximite_condictio_et_GitHub_Marketplace·_Actions_to_improve_your_workflow.png</span><span class="informations">1046×957 107 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Sorry to tell you that currently you could not add more information when a task fails to the “Annotation” popin. But we are looking into ways to improve annotation experience!</p>
<p>In addition, github actions provide a way to set a warning or error message , these warning and error message could show on annotations. Please refer to: <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#set-a-warning-message-warning" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#set-a-warning-message-warning  </a></p>
